File: Interface Gráfica - Tela de Criação de Herói/Java/Chefe.java
class Chefe extends Personagem {

    public Chefe(String nome) {
        super(nome, 200, 20);
    }

    @Override
    public void ficha() {
        System.out.printf("%n[CHEFE]:%n");
        super.ficha();

    }

    @Override
    public String habilidade() {
        return nome + " usa Ataque devastador";
    }

}

File: Exceções - IF Quest/Python/chefe.py
from personagem import Personagem


class Chefe(Personagem):
    def __init__(self, nome):
        super().__init__(nome, 200, 20)

    def ficha(self):
        print("\n[CHEFE]:")
        super().ficha()

    def habilidade(self):
        return f"{self.nome} usa Ataque devastador"
